worlds/shorthike/docs/setup_en.md

```diff
@@ -0,0 +1,33 @@
+# A Short Hike Multiworld Setup Guide
+
+## Required Software
+
+- A Short Hike: [Steam](https://store.steampowered.com/app/1055540/A_Short_Hike/)
+    - The Epic Games version of A Short Hike will also work.
```

Comment:
```suggestion
    - The Epic Games Store version of A Short Hike will also work.
```

Reply:
I run this mod on the DRM free version from itch.io, and it's been working flawlessly. Many people were given a copy of this from the itch.io Racial Justice & Palestinian Aid bundles (from 2020 and 2021, respectively), so that's a nontrivial ownership source. Unless you have intent to not "support" this version, the documentation should probably also acknowledge that?